src/api-types.ts

```diff
@@ -1040,3 +1040,27 @@ export interface LastEditedTimePropertySchema {
 export interface LastEditedByPropertySchema {
   last_edited_by: Record<string, never>
 }
+
+/*
+ * Update database property schema (input)
+ */
+export interface RenamePropertySchema {
+  name: string
+}
+
+type UpdateSelectOptionSchema = SelectOptionSchema | SelectOption
+
+export interface UpdateSelectPropertySchema {
+  select: { options?: UpdateSelectOptionSchema[] }
```

Comment:
the `options` field is nullable? what's the behavior when an empty object is passed into `select`?

Reply:
if it's undefined, then there will be no options (similar to when you create a "select" property in the client before adding any options). In [create database](https://developers.notion.com/reference/create-a-database#select-configuration), the `options` are optional (ha).